services/client/src/components/LeaderboardTable.tsx

```diff
@@ -77,24 +84,37 @@ const LeaderboardTable = ({ data }: Props) => {
         </tbody>
       </table>
       {filteredData.length > 0 && (
-        <div className="flex flex-row justify-end items-center bg-blue-dark text-white rounded-b-md">
-          <button
-            className="px-5 py-1 text-xl disabled:text-gray-500"
-            onClick={() => changePage(page - 1)}
-            disabled={page === 1}
-          >
-            &#x25C0;
-          </button>
-          <div>
-            Page {page} of {totalPages}
+        <div className="flex flex-row justify-between items-center px-1 bg-blue-dark text-white rounded-b-md">
+          <div className="flex flex-row items-center">
+            <label className="mx-5">
+              <input
+                className="accent-green h-3 w-3 mr-2"
+                type="checkbox"
+                checked={showFails}
+                onChange={(e) => setShowFails(e.target.checked)}
+              />
+              Show fails
```

Comment:
Suggestion: Rename to state "Show Failed Missions", instead.

Reply:
updated:
<img width="1394" alt="image" src="https://user-images.githubusercontent.com/33457590/171878811-aa4cc032-bc31-4309-b754-bc17cd0658d0.png">